src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -135,6 +143,9 @@ private[query] class EventsByTagPublisher(
 
     case Cancel =>
       context.stop(self)
+
+    case tagWritten: String if tagWritten == tag =>
+      self ! Continue
```

Comment:
yes, it would be interesting to share the live queries, but that is probably a slightly more complicated task, and perhaps a separate PR